toolchain/base/canonical_value_store.h

```diff
@@ -29,6 +29,7 @@ class CanonicalValueStore {
   using ValueType = ValueStoreTypes<ValueT>::ValueType;
   using RefType = ValueStoreTypes<ValueT>::RefType;
   using ConstRefType = ValueStoreTypes<ValueT>::ConstRefType;
+  using IdType = IdT;
```

Comment:
Nit: I'd expect this to be above `KeyType`, to match the order of the template parameters.

Reply:
3399aa7e98a10a04a9c688cf5f56ce61b16297a5